CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,338 @@
+# 1. Code of Conduct for the Open Research Community Building Team
+
+> _A code of conduct is a set of rules outlining the norms, rules, and responsibilities or proper practices of an individual party or an organization._
+
+This Code of Conduct document outlines our expectations for 
+participants within the team as well as this online repository (and the ideas, projects, and collaborations that might emerge from it).
+We also provide steps to reporting any concern, uncomfortable situation 
+or unacceptable behaviour by other participants. 
+We are committed to providing a welcoming, collaborative and inspiring experience 
+for all members of this team and expect our code of conduct to be honoured.
+
+## 2. Our Commitment
+
+We as members, contributors, and leaders commit to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We commit to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## 3. Restorative Practice
+
+We want to establish a Code of Conduct that focuses on creating a welcoming and safe environment for
+everyone involved.
+If a participant does not abide by the Code of Conduct, we will use the 
+“restorative practices” approach to handle the situation. Restorative Practices has 
+their roots in “restorative justice,” a way of looking at wrongdoing that focuses on 
+repairing the harm done to people and the relationships rather than on punishing offenders or wrongdoers. 
+
+## 4. Our Standards
+
+- 4.1 **Be friendly and patient.**
+
+- 4.2 **Be welcoming**: we strive to build a welcoming and supportive team.
+  As community facilitators, we also strive to apply this value in creating 
+  Communities of Practice that welcomes and supports members of all backgrounds 
+  and identities. This includes, but is not limited to
+  members of any race, ethnicity, culture, national origin, colour, immigration
```

Comment:
The 'commitment' section is supposed to be exhaustive. This one is not exhaustive, instead we highlight the top protected characteristics. This can be kept identical if you suggest.

Reply:
No, that's okay! I would maybe just remove "colour" from this list as I think it doesn't add anything to race and reads as perhaps older terminology? Do let me know if I'm missing something though 😊

Reply:
Now that inconsistency is definitely bothering me. Let me suggest some consistent change.

Reply:
I have merged "our commitment" with "diversity statement" into single clause 2: https://github.com/alan-turing-institute/open-community-building/blob/code-of-conduct/CODE_OF_CONDUCT.md#2-our-commitment-and-diversity-statement

To specifically address the inconsistency issue, I have now updated 4.2 that references clause 2.
Let me know if this actually works better.

Reply:
I like that!

Reply:
Great! Thank you Eirini.